daemon/daemon.go

```diff
@@ -611,16 +610,16 @@ func (d *Daemon) doReboot(sigCh chan<- os.Signal, rst restart.RestartType, immed
 	if err != nil {
 		return err
 	}
-	action := rebootReboot
+	action := boot.RebootReboot
 	switch rst {
 	case restart.RestartSystemHaltNow:
-		action = rebootHalt
+		action = boot.RebootHalt
 	case restart.RestartSystemPoweroffNow:
-		action = rebootPoweroff
+		action = boot.RebootPoweroff
 	}
 	// ask for shutdown and wait for it to happen.
 	// if we exit snapd will be restared by systemd
-	if err := reboot(action, rebootDelay); err != nil {
+	if err := reboot(action, rebootDelay, nil); err != nil {
```

Comment:
this should probably receive a rebootInfo (that's why I'm asking about capturing above) ?